cmd/reset.go

```diff
@@ -72,11 +73,10 @@ func init() {
 
 // parseConfigEntryKubefirstChecks gathers the kubefirst-checks section of the Viper
 // config file and parses as a map[string]bool
-func parseConfigEntryKubefirstChecks(raw interface{}) (map[string]bool, error) {
-	if raw == nil {
+func parseConfigEntryKubefirstChecks(checks map[string]interface{}) (map[string]bool, error) {
```

Comment:
This was an optimization pointed out by `golangci-lint`: there's only one caller and the type is known, so it doesn't need an interface.